diff -Nur --exclude *~ ipchains-1.3.9/Makefile ipchains-1.3.9.new/Makefile
--- ipchains-1.3.9/Makefile	Tue Oct 20 07:15:29 1998
+++ ipchains-1.3.9.new/Makefile	Sat Jun  9 12:37:20 2001
@@ -29,7 +29,7 @@
 all:		ipchains ipfw.4 ipchains.8
 
 ipchains:	libipfwc/libipfwc.a libipfwc/libipfwc.h ipchains.c libipfwc/ipfwc_kernel_headers.h
-	$(CC) $(CFLAGS) -o ipchains ipchains.c libipfwc/libipfwc.a
+	$(CC) $(CFLAGS) $(LDFLAGS) -o ipchains ipchains.c libipfwc/libipfwc.a $(LDLIBS)
 
 install:	ipchains ipfw.4 ipchains.8
 	@-mv $(SBIN)/ipchains $(SBIN)/ipchains.old
