diff -uNr ipchains-1.3.10.orig/ipchains.c ipchains-1.3.10/ipchains.c
--- ipchains-1.3.10.orig/ipchains.c	Fri Sep  1 13:33:04 2000
+++ ipchains-1.3.10/ipchains.c	Tue Mar  5 18:22:25 2002
@@ -1711,18 +1711,6 @@
 	}
 	else ret = 0;
 
-	/* Warn if there are non alpha-numerics in interface name. */
-	if (warnings) {
-		for (i = 0; vianame[i]; i++) {
-			if (!isalnum(vianame[i])) {
-				printf("Warning: wierd character in interface"
-				       " `%s' (No aliases, :, ! or *).\n",
-				       vianame);
-				break;
-			}
-		}
-	}
-
 	return ret;
 }
 
